webpages/styles/colors.css

```diff
@@ -18,20 +18,23 @@
   --dark-red-text: var(--dark-red-variant);
   --white-text: #fff;
 
-  --page-background: #222;
-  --content-background: #2a2a2a;
+  --page-background: #181818;
+  --content-background: #272727;
   --navigation-background: #333;
-  --search-toolbar-background: #2a2a2a;
-  --button-background: #222;
+  --search-toolbar-background: #1c1c1c;
+  --button-background: #181818;
   --button-hover-background: #1a1a1a;
-  --input-background: #222;
-  --search-input-background: #111;
+  --tab-hover-underline: #1e1e1e;
+  --input-background: #1d1d1d;
+  --search-input-background: #272727;
   --switch-border: #808080;
   --switch-inner-background: #eee;
   --tooltip-background: #000;
-  --comment-background: #222;
-  --unread-comment-background: #000;
-  --notice-background: var(--navigation-background);
+  --comment-background: #1c1c1c;
+  --admin-comment-background: #0000007a;
+  --unread-comment-background: #0d0d0d;
+  --unread-comment-border: inset 0px 0px 0px 2px #161616;
+  --notice-background: var(--yellow-opacity50);
```

Comment:
This doesn't look great on the dark background.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/02d2e350-3687-414d-bf3e-f358c37fa904)

Reply:
hmm

Reply:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/7691d9b8-c3de-43f1-a9b6-2739006a6ef8)
wbu this?

Reply:
That's fine unless you want to just use `--yellow`.